lib/turbo/broadcastable/test_helper.rb

```diff
@@ -155,10 +155,13 @@ def assert_no_turbo_stream_broadcasts(stream_name_or_object, &block)
       #     assert_equal "replace", remove["action"]
       #
       def capture_turbo_stream_broadcasts(stream_name_or_object, &block)
-        block&.call
-
         stream_name = stream_name_from(stream_name_or_object)
-        payloads = broadcasts(stream_name)
+
+        if block_given?
+          payloads = new_broadcasts_from(broadcasts(stream_name), stream_name, "capture_turbo_stream_broadcasts", &block)
+        else
+          payloads = broadcasts(stream_name)
+        end
```

Comment:
Could this be flattened to a conditional assignment?

```suggestion
        payloads =
          if block_given?
            new_broadcasts_from(broadcasts(stream_name), stream_name, "capture_turbo_stream_broadcasts", &block)
          else
            broadcasts(stream_name)
          end
```

Reply:
Sure! I have no strong preference here.

Updated